lib/topological_inventory/persister/worker.rb

```diff
@@ -98,8 +98,7 @@ def queue_opts
       def default_messaging_opts
         {
           :protocol   => :Kafka,
-          :client_ref => "persister-worker",
-          :group_ref  => "persister-worker",
+          :client_ref => "persister-worker-#{Time.now.to_i}",
```

Comment:
@slemrmartin how about appending the hostname as opposed to a timestamp? It accomplishes the same but gives us more insight as to what pod is connecting.

Reply:
done